### DISCUSSION TITLE
UbuntuMerge?

### DISCUSSION THREAD

**Discussion by rolfeanderson** [WinMerge/winmerge#617] [Q&A] [answered]
Is there a program similar to WinMerge that can be used on Ubuntu?

**Comment by sdottaka** [comment, marked as answer]
[Meld](https://meldmerge.org)
[KDiff3](https://invent.kde.org/sdk/kdiff3)

I think the following will be helpful.

https://en.wikipedia.org/wiki/Comparison_of_file_comparison_tools

**Reply by Gitoffthelawn** [reply]
> I think the following will be helpful.
> 
> https://en.wikipedia.org/wiki/Comparison_of_file_comparison_tools

I'm not sure that comparison table meets my needs.  I'm holding out for a comparison table of comparison tables of comparison tools.  ;)
